packages/tools/src/tools/annotation/LivewireContourTool.ts

```diff
@@ -72,6 +67,43 @@ class LivewireContourTool extends ContourSegmentationBaseTool {
       supportedInteractionTypes: ['Mouse', 'Touch'],
       configuration: {
         preventHandleOutsideImage: false,
+        /**
+         * Configuring this to a value larger than 0 will snap handles to nearby
+         * livewire points, within the given rectangle surrounding the clicked point.
+         */
+        snapHandleNearby: 1,
```

Comment:
I see, maybe add more documentation that if 0 it means it will add the handle right at the clicked point which might not be at the edge

Reply:
pleaes mention this is pixel unit